### DISCUSSION TITLE
Programación en las cosas

### DISCUSSION THREAD

**Discussion by imlianeth** [hackatomik/SISCOx256#46] [Chill | Carrera]
¿Creen que en el futuro la mayoria de cosas sea programable, o qué sería lo minimo qué podriamos programar?

**Comment by quantumquirkxyz** [comment]
En el futuro, es probable que la mayoría de las cosas sean programables debido al avance continuo de la tecnología y la integración de sistemas inteligentes en nuestra vida diaria. Desde electrodomésticos y automóviles hasta dispositivos portátiles y sistemas de infraestructura, la capacidad de programar estos elementos ofrecerá más control, personalización y eficiencia. La programación permitirá que estos dispositivos se adapten a nuestras necesidades específicas, automatizando tareas y mejorando la interacción entre el usuario y la tecnología.

En cuanto a lo mínimo que podríamos programar, incluso los objetos más simples pueden ser programables en un futuro cercano. Esto podría incluir cosas como electrodomésticos básicos (refrigeradores que gestionen inventarios), sistemas de iluminación (luces que se ajusten automáticamente según la hora del día o la presencia de personas), o dispositivos de entretenimiento (televisores que recomienden contenido basado en el gusto del usuario). La clave es que, a medida que la tecnología se vuelve más accesible y asequible, cada vez más aspectos de nuestra vida cotidiana podrán ser programados para hacerlos más inteligentes y útiles.

**Reply by imlianeth** [reply]
Es muy interesante saber las cosas que en algún momento podremos programar en el futuro. 
Tal vez un ejemplo de lo minímo podría ser alexa o un echo dot, que ya tienen la capacidad de programar la hora de encendido o apagado de luces, entre muchas otras funciones.

**Reply by quantumquirkxyz** [reply]
Aunque no lo creas, hasta ya puedes reprogramarlas o incluso crear tu propia asistente virtual ¿JARVIS?

**Reply by quantumquirkxyz** [reply]
https://youtu.be/ImOR0o-QHOQ?si=86QUSLnoQ4JW7aoa
